hbase-thrift/src/test/java/org/apache/hadoop/hbase/thrift/TestThriftSpnegoHttpFallbackServer.java

```diff
@@ -0,0 +1,238 @@
+/*
+ * Copyright The Apache Software Foundation
```

Comment:
No copyright statements in file headers please.

Reply:
Hmmm ok copied from another file nearby this one will remove.